Sources/Networking/Core/APIManager.swift

```diff
@@ -37,7 +44,13 @@ open class APIManager: APIManaging, Retryable {
         errorProcessors: [ErrorProcessing] = []
     ) {
         /// generate session id in readable format
-        sessionId = Date().ISO8601Format()
+        if #unavailable(iOS 15) {
```

Comment:
Maybe this could be reused? I see it three times with the same exact code.

Reply:
yes but original it is one line command. It will be back soon I hope